scripts/update_rif_table.py

```diff
@@ -0,0 +1,133 @@
+# Copyright (C) University of Tennessee Health Science Center, Memphis, TN.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# This program is available from Source Forge: at GeneNetwork Project
+# (sourceforge.net/projects/genenetwork/).
+#
+# Contact Drs. Robert W. Williams and Xiaodong Zhou (2010)
+# at rwilliams@uthsc.edu and xzhou15@uthsc.edu
+#
+#
+#
+# This module is used by GeneNetwork project (www.genenetwork.org)
+#
+# Created by GeneNetwork Core Team 2010/08/10
+# Updated on Lei Yan 2011/02/08
+# created by Lei Yan 02/08/2011
+
```

Comment:
*suggestion (non-blocking):* Consider removing this front-matter.

Some of the information here is old, like the URL link to the sourceforge instance.  We already have the main licensing info in the main repo as well.

*suggestion (non-blocking):* Consider having:

```
#! /usr/bin/env python3
```

at the very top of this script.  That way, when running "./update_rif_table.py" from the terminal, your terminal emulator can automatically pick python3 without the need for you explicitly provide it.

Reply:
Cool. I've dropped the front-matter and added the shebang.